### ISSUE TITLE
[ADD] 220v Opto Sensor and Relay to control 220v

### ISSUE THREAD

**Issue by juanpintom** [juanpintom/Souliss_ESP_Examples#30]
Add to pins 4 and 5 an option to read state from a 220v line. An opto AC will be used. It works in inverted mode so it return 3.3v when 220v its powered off (0V) and return 0v when 220v line its turned ON.

This state will define the state of a T11 to know if 220v it's ON/OFF, and with a Relay we will switch this state.

Schemes and more explanation it's needed, it's only on my mind now xD

**Comment by marksev1** [comment]
Are you planning on making a custom pcb for 220V operation?  Perhaps a Juan's Esp-220V-switch or a Juan's Esp-220V-power socket :). 

Since now you have sensors and leds covered, only 220V is left :D

**Comment by juanpintom** [comment]
Yes, it will be a separate board to connect on PINS 4,5 of the REV2 pcb, so it will be an option to add like a plugin module an will be configurable via webinterface.
